pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -299,24 +295,64 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 	// attempt processing again later. This could have been caused by a
 	// temporary network failure, or any other transient reason.
 	if err != nil {
-		c.recorder.Event(etcdstorage, corev1.EventTypeWarning, ErrUnknown, err.Error())
+		etcdstorage, updateErr := c.updateEtcdStorageStatus(etcdstorage, etcdstoragev1alpha1.EtcdStorageCondition{
+			Type:   etcdstoragev1alpha1.Deployed,
+			Status: etcdstoragev1alpha1.ConditionFalse,
```

Comment:
Be careful here.  It's not always false.  It could be unknown, right?  If the get fails, it may be there, you just can't see it.